CONTRIBUTING.md

```diff
@@ -19,6 +19,20 @@ If you have any questions, contact us on our [Slack Channel](https://mizar-group
 
 ## Contributing
 
+### Contributor Workflow
+
+Please do not ever hesitate to ask a question or send a pull request.
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where to base the contribution. This is usually master.
+- Make commits of logical units.
+- Make sure commit messages are in the proper format (see below).
+- Push changes in a topic branch to a personal fork of the repository.
+- Submit a pull request to [mizar](https://github.com/futurewei-cloud/mizar).
+- The PR must receive an approval from two maintainers.
```

Comment:
Minor: receive approval from two maintainers.

Reply:
As agreed, we will change to two members and one maintainer.